client/signup/steps/user/index.jsx

```diff
@@ -52,13 +51,21 @@ export default React.createClass( {
 	},
 
 	submitForm( form, userData, analyticsData ) {
+		var abtest = require( 'lib/abtest' ).abtest;
```

Comment:
I'd import this at the top of the file under internal dependencies:

`import { abtest } from 'lib/abtest';`